tests/console/sshd.pm

```diff
@@ -136,6 +141,37 @@ sub run {
     clear_console if !is_serial_terminal;
 }
 
+sub test_cryptographic_policies() {
+    my %args        = @_;
+    my $remote_user = $args{remote_user};
+
+    my @crypto_params = (["Ciphers", "cipher", "-c "], ["HostKeyAlgorithms", "key", "-o HostKeyAlgorithms="], ["KexAlgorithms", "kex", "-o kexalgorithms="], ["MACS", "mac", "-m "]);
+    my @policies;
+
+    # Create an array of the different cryptographic policies that will be tested
+    for my $i (0 .. $#crypto_params) {
+        my $obj = ssh_crypto_policy->new(name => $crypto_params[$i][0], query => $crypto_params[$i][1], cmd_option => $crypto_params[$i][2]);
+        push(@policies, $obj);
+    }
+
+    # Add all available algorithms to sshd_config
+    foreach my $policy (@policies) {
+        $policy->add_to_sshd_config();
+    }
+
+    my $sshd_config = script_output("cat /etc/ssh/sshd_config");
+    record_info("Restart sshd", $sshd_config);
+    systemctl("restart sshd");
+
+    # Add all the ssh public key hashes as known hosts
+    assert_script_run("ssh-keyscan -H localhost >> ~/.ssh/known_hosts");
```

Comment:
As there are only `localhost` keys in ~/.ssh/known_hosts` an you rescan all of them I'd do
````suggestion
    assert_script_run("ssh-keyscan -H localhost > ~/.ssh/known_hosts");
```